### PR TITLE
pinit:0.1.0

### PR DESCRIPTION
**Pull request by OrangeX4**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Pin things as you like, especially useful for creating slides.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

**Comment by reknih** [comment]
Thank you for your package submission! We have to ask you to change the name of this package because it is not save for work (determined by Google search and Wikipedia article; see **Safety** package guideline in the Readme). We can go forward with merging this as soon as the name is changed!

**Comment by OrangeX4** [comment]
I have renamed it with `pinit` :-)

File: packages/preview/pinit/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "pinit"
+version = "0.1.0"
+authors = ["OrangeX4"]
+entrypoint = "lib.typ"
+exclude = ["./examples", "./README.md"]
```
Comment (laurmaedje) [inline packages/preview/pinit/0.1.0/typst.toml:6]:
```suggestion
exclude = ["./examples"]
```
Some existing packages exclude the README, but the key was only intended for large support files.

Comment (OrangeX4) [reply packages/preview/pinit/0.1.0/typst.toml]: OK, fixed it.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**